app/models/planning_application.rb

```diff
@@ -440,6 +440,10 @@ def has_policy_class?(section)
     policy_classes.pluck(:section).include?(section)
   end
 
+  def planning_history_enabled?
```

Comment:
I'm not sure it makes sense to define this in the planning application model? If its dependent on an env var it won't be planning application specific.

Reply:
@allbecauseyoutoldmeso yeah I was thinking of moving it somewhere else, but as as I couldn't find a good name and this is the first case of a feature flag, I thought of adding it on the planning_application model 😂 

Happy to change it on a separate PR or when we will have more feature flags 😄